modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/file_entry_history.jsp

```diff
@@ -69,7 +76,25 @@
 		</li>
 
 	<%
+		index++;
 	}
 	%>
 
+	<c:if test='<%= FeatureFlagManagerUtil.isEnabled("LPS-175915") && (index >= fileVersionsLimit) %>'>
```

Comment:
if my previous comment make sense : instead of index is `fileEntry.getFileVersionsCount(status)`